src/hiveboard_bridge/src/main.cpp

```diff
@@ -38,7 +36,7 @@ int main(int argc, char** argv) {
     MessageHandler messageHandler;
 
     // Register callbacks
-    CallbackFunction moveByCallback = [&](CallbackArgs args, int argsLength) {
+    CallbackFunction moveByCallback = [&](CallbackArgs args, int argsLength, CallbackContext ctx) {
```

Comment:
Idk if it would be better to pass a smart pointer to the message, allowing us to have all the information if we add more fields, (though then you need to do all the variant checking again). It would require to use a new for the message though, but that isn't be a problem on ROS

Reply:
I think this is maybe a bit overkill for the user. I would rather provide the simplest interface possible to the user and hide the checks as much we can. From the user stand point, the only info necessary to answer to function calls are a few Ids.